src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -162,6 +162,7 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
   }
 
   onMutationError(errors, errorModalTitle?, errorModalMessage?) {
+    console.error("Shipping/index.tsx", errors)
```

Comment:
I'd accidentally removed this in a previous PR, but these log messages are super helpful for debugging.

Reply:
Sooner rather than later we should drop in a logger similar to what we have in prediction, which hooks right into sentry on `logger.error` (something I'd like to look at next quarter)